packages/backend/src/models/GroupsModel.ts

```diff
@@ -226,31 +228,41 @@ export class GroupsModel {
         createGroup: CreateGroup & { organizationUuid: string },
     ): Promise<GroupWithMembers> {
         const groupUuid = await this.database.transaction(async (trx) => {
-            const results = await trx.raw<{
-                rows: { created_at: Date; group_uuid: string }[];
-            }>(
-                `
+            try {
+                const results = await trx.raw<{
+                    rows: { created_at: Date; group_uuid: string }[];
+                }>(
+                    `
                     INSERT INTO groups (name, organization_id)
                     SELECT ?, organization_id
                     FROM organizations
                     WHERE organization_uuid = ? RETURNING group_uuid, groups.created_at
                 `,
-                [createGroup.name, createGroup.organizationUuid],
-            );
-            const [row] = results.rows;
-            if (row === undefined) {
-                throw new UnexpectedDatabaseError(`Failed to create group`);
-            }
-
-            if (createGroup.members && createGroup.members.length > 0) {
-                await GroupsModel.addGroupMembers(
-                    trx,
-                    row.group_uuid,
-                    createGroup.members.map((m) => m.userUuid),
+                    [createGroup.name.trim(), createGroup.organizationUuid],
```

Comment:
I noticed that we were allowing trailing whitespace in group names so I'm stripping it out here before save.